.github/workflows/linux.yml

```diff
@@ -162,7 +165,22 @@ jobs:
         fi
 
         if [ -n "${VCPKG_BINARY_SOURCES}" ]; then
-          sed -i "/build-environment:/a \    - VCPKG_BINARY_SOURCES: ${VCPKG_BINARY_SOURCES}" snap/snapcraft.yaml
+          sed -i '/build-environment:/ {
```

Comment:
Not related to this PR, but this whole sed'ing and patch'ing of the YAML file makes me wonder if we should switch to using an inline Python script to modify the YAML instead.

Reply:
Yeah, the `sed` language isn't the most pleasant to work with. Let me know how strongly you feel about it and if you would like to see it changed here.